packages/expo-modules-core/ios/AppDelegates/ExpoAppDelegate.swift

```diff
@@ -18,10 +18,25 @@ open class ExpoAppDelegate: UIResponder, UIApplicationDelegate {
   @objc
   public let reactDelegate = ExpoReactDelegate(handlers: reactDelegateHandlers)
 
+    @objc
+    open override func buildMenu(with builder: UIMenuBuilder) {
+        let parsedSubscribers = subscribers.filter {
+            $0.responds(to: #selector(buildMenu(with:)))
+        }
+
+        if parsedSubscribers.isEmpty {
+            return
+        }
+
+        parsedSubscribers.forEach {
+            $0.buildMenu(with: builder)
+        }
```

Comment:
I think this 👇 implementation would look a bit better. Could you also make sure your code use 2-space indent width?
```swift
subscribers
  .filter { subscriber in
    return subscriber.responds(to: #selector(buildMenu(with:)))
  }
  .forEach { subscriber in
    subscriber.buildMenu(with: builder)
  }
```